quoteit.py

```diff
@@ -15,82 +15,109 @@
 from requests import exceptions
 
 
-############################################################################
 class Comments:
-   
-
     def __init__(self, r):
-        # subreddit to parse through
-        # set to /r/all, but could be
-        # set to a specific sub if needed
-        # r is the praw Reddit Object
+        """
+        subreddit to parse through set to /r/all, but could be set to a
+        specific sub if needed
+
+        'r' is the praw Reddit Object
+        """
         self.r = r
         self.db = Database()
-        self.regex = re.compile('quoteit! ("[\s\S]*")*[\s-]*([\/u\/]*[\w_\' -]*)',
-                                flags = re.IGNORECASE | re.UNICODE)
+        self.regex = re.compile(
+            """
+                # QuoteIt! "[quote]" /u/[username]
+                # QuoteIt! '[quote]' - u/[username]
+                # etc
+
+                quoteit! \s+
```

Comment:
Would prefer to have <code>\s*</code> to still catch a phrase if someone entered <code>'QuoteIt!"test 123" /u/someone</code>